youtube_dl/extractor/pornhub.py

```diff
@@ -297,16 +297,17 @@ def parse_quality_items(js_str):
                     add_video_url(definition.get('url'))
 
         if not video_urls:
-            FORMAT_PREFIXES = ('media', 'quality')
+            FORMAT_PREFIXES = ('media', 'quality', 'qualityItems')
             js_vars = extract_js_vars(
                 webpage, r'(var\s+(?:%s)_.+)' % '|'.join(FORMAT_PREFIXES),
                 default=None)
             if js_vars:
                 for key, format_url in js_vars.items():
-                    if key.startswith('qualityItems'):
-                        parse_quality_items(format_url)
                     if any(key.startswith(p) for p in FORMAT_PREFIXES):
-                        add_video_url(format_url)
+                        if key.startswith('qualityItems'):
+                            parse_quality_items(format_url)
+                        else:
```

Comment:
This is not equivalent, not matching `qualityItems` does not mean it will match `media` or `quality`.